pgjdbc/src/main/java/org/postgresql/jdbc/PgDatabaseMetaData.java

```diff
@@ -2696,7 +2696,7 @@ public ResultSet getUDTs(@Nullable String catalog, @Nullable String schemaPatter
       long longTypOid = typeInfo.intOidToLong(typOid);
       int sqlType = typeInfo.getSQLType(typOid);
 
-      sqlwhen.append(" when t.oid = ").append(longTypOid).append(" then ").append(sqlType);
+      sqlwhen.append(" when oid = ").append(longTypOid).append(" then ").append(sqlType);
```

Comment:
Is it possible to use an alias for `oid=..` condition?
The current syntax is obscure in the sense that the source of `oid` column is hard to tell.

Reply:
I like that idea. Making the intent clear is inexpensive

Reply:
@vlsi, sorry just saw your suggestion. 
I can add the alias and submit another PR.